addons/web/static/tests/views/form/form_view.test.js

```diff
@@ -1193,7 +1194,6 @@ test(`invisible, required and readonly using company evalContext`, async () => {
             country_code: "AR",
         },
     ];
-    cookie.set("cids", "3-1");
```

Comment:
just for me to be sure that I understand. Here we could have patched the user right?